langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -41,9 +41,9 @@ public type Info readonly & InfoInternal;
 
 # An instant in time.
 public type Timestamp readonly & object {
-    # Returns milliseconds since 1970-01-01T00:00:00Z, not including leap seconds
+    # + return - milliseconds since 1970-01-01T00:00:00Z, not including leap seconds
```

Comment:
We need the generic documentation also. Shall we keep both for now?

```suggestion
    # Returns milliseconds since 1970-01-01T00:00:00Z, not including leap seconds
    # + return - milliseconds since 1970-01-01T00:00:00Z, not including leap seconds
```